OnionBrowser/Browsing/TabCell.swift

```diff
@@ -25,6 +25,7 @@ class TabCell: UICollectionViewCell, UIGestureRecognizerDelegate {
     @IBOutlet weak var title: UILabel!
 
     @IBOutlet weak var container: UIView!
+	@IBOutlet weak var preview: UIImageView!
```

Comment:
Fix indentation. Project standard is tabs, 4 spaces wide.

Reply:
Oh, ha. I see, this is because other lines in the code have broken indentation. Sorry.